main.cpp

```diff
@@ -1,4 +1,4 @@
-#include "tfclone.h"
+#include "buildTensorflow.h"
```

Comment:
I think we need to come up with a good name quickly :)

Reply:
Haha yes !